compute/oslogin/requirements.txt

```diff
@@ -2,5 +2,5 @@ google-api-python-client==2.47.0
 google-auth==2.6.2
```

Comment:
Dependency conflict with google-auth version needs to be fixed before this can be merged. Reassigning to @m-strzelczyk as he is working on oslogin test issues